src/components/MemoAddForm.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <div class="backdrop" @click.self='closeAddForm'>
+    <div class="modal">
+      <h2>Memo Add</h2>
+        <label>
+          <input type='text' placeholder="Title" v-model='title'>
+        </label>
+        <label>
+          <textarea placeholder="Content" v-model='content'></textarea>
+        </label>
+        <button @click='addMemo()'>Add</button>
```

Comment:
引数なしで method を呼び出すのであれば `()` はなくても良いかもです。

```suggestion
        <button @click='addMemo'>Add</button>
```

Reply:
``()``を削除しました！